documentation/modules/ROOT/pages/connectors/jdbc.adoc

```diff
@@ -495,56 +495,101 @@ If the database does not support time or timestamps with time zones, the mapping
 
 .Mappings between {prodname} dialect-specific logical types and column data types
 |===
```

Comment:
```suggestion
[cols="as,as,as,as,as,as",options="header"]
|===
```
Played around with this locally and it seems to render things as we would like. The `a` applies asciidoc block formatting to each column cell, and the `s` renders cell contents in a smaller font, which provides a better fit. TBH, I'm not 100% sure that what works locally will render the same in the published docs.